hs_core/templates/pages/genericresource.html

```diff
@@ -830,14 +830,13 @@ <h4 class="modal-title">Manage access</h4>
 
         {#        ======= Content =======#}
 
+        {% if cm.resource_type != "ToolResource" %}
```

Comment:
@mauriel133 Although this can achieve what is needed for this issue, I think it is probably better to provide a context variable, say, show_content_files, indicating whether to show content files or not. By default, show_content_files will be true, but specific resource types such as ToolResource can set it to false as needed. I think this will result in more extensible code since more use cases might come up over time which requires not showing content files. Adding @ironfroggy, @pkdash, and @selimnairb into the loop to chime in on this.